packages/js/src/general/initialize.js

```diff
@@ -102,15 +102,29 @@ domReady( () => {
 			isConsentGranted: false,
 		},
 		isRedirectedFromSiteKit: false,
+		storagePrefix: "",
+		yoastVersion: "",
+		widgetsCacheTtl: {},
 	} );
 
 	const remoteDataProvider = new RemoteDataProvider( { headers } );
+
 	const dataProvider = new DataProvider( { contentTypes, userName, features, endpoints, headers, links, siteKitConfiguration } );
 	const dataFormatters = {
 		comparisonMetricsDataFormatter: new ComparisonMetricsDataFormatter( { locale: userLocale } ),
 		plainMetricsDataFormatter: new PlainMetricsDataFormatter( { locale: userLocale } ),
 	};
 
+	const remoteCachedDataProviders = Object.entries( siteKitConfiguration.widgetsCacheTtl ).reduce( ( providers, [ key, value ] ) => {
```

Comment:
So because you save it in the `siteKitConfiguration` no other widget will have cache? Just Site Kit ones 🤔

Reply:
It's entirely possible that no other widgets will have that kind of cache. 

But in any case, we'll cross that bridge when we get there, I dont see many issues with refactoring that once we actually need to.

Reply:
Added a separate layer for creating the browser cache configuration, done [here](https://github.com/Yoast/wordpress-seo/pull/22186/commits/4ea81ea66971ddc812975ab4b7a2ac56b2c601a1).